airbyte-integrations/connectors/source-sentry/integration_tests/abnormal_state.json

```diff
@@ -31,5 +31,16 @@
         "name": "events"
       }
     }
+  },
+  {
+    "type": "STREAM",
```

Comment:
Please also add this stream to `integration_tests/configured_catalog.json` as it is used in the incremental acceptance test